plugins/woocommerce/includes/admin/helper/class-wc-helper-admin.php

```diff
@@ -39,11 +39,13 @@ public static function add_marketplace_settings( $settings ) {
 		$auth_user_data  = WC_Helper_Options::get( 'auth_user_data', array() );
 		$auth_user_email = isset( $auth_user_data['email'] ) ? $auth_user_data['email'] : '';
 
+		$store_location = wc_get_base_location();
 		$settings['wccomHelper'] = array(
 			'isConnected' => WC_Helper::is_site_connected(),
 			'connectURL'  => self::get_connection_url(),
 			'userEmail'   => $auth_user_email,
 			'userAvatar'  => get_avatar_url( $auth_user_email, array( 'size' => '48' ) ),
+			'storeCountry' => $store_location['country'],
```

Comment:
Is it possible to inline `wc_get_base_location()` call? Not a big issue, just curious.

```suggestion
		$settings['wccomHelper'] = array(
			'isConnected' => WC_Helper::is_site_connected(),
			'connectURL'  => self::get_connection_url(),
			'userEmail'   => $auth_user_email,
			'userAvatar'  => get_avatar_url( $auth_user_email, array( 'size' => '48' ) ),
			'storeCountry' => wc_get_base_location()['country'],
```

Reply:
Yes, I prefer this. I tried it, but for some reason it seemed less clear to me. Looking at it now, that's far better.